packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.stories.js

```diff
@@ -45,18 +55,22 @@ const Template = ({
   <StoryContext
     theme={{ ...combinedTheme, dark }}
   >
-    <ChoiceButton
-      choiceId={choiceId}
-      choiceLabel={choiceLabel}
-      disabled={disabled}
-      hasFocus={hasFocus}
-      onChoose={() => console.log(choiceId)}
-      onKeyDown={({ key }) => console.log(choiceId, key)}
-      selected={selected}
-      src={src}
-      tabIndex={tabIndex}
-      thumbnailSize={thumbnailSize}
-    />
+    <Box
+      width={getColumnWidth(thumbnailSize)}
+    >
+      <ChoiceButton
+        choiceId={choiceId}
+        choiceLabel={choiceLabel}
+        disabled={disabled}
+        hasFocus={hasFocus}
+        onChoose={() => console.log(choiceId)}
+        onKeyDown={({ key }) => console.log(choiceId, key)}
```

Comment:
These should use [Storybook actions](https://storybook.js.org/docs/react/essentials/actions) rather than console logs.

Reply:
yes! nice catch.
I still plan to rewrite all the stories and create a full survey task story, opened #3816 to track.